### PR TITLE
Merge Mol 49 into dev

### PR DESCRIPTION
**Pull request by kdm5498**
### Description
Contains the components needed to perform the visualization of a docked ligand and protein. For a test upload file, see file in JIRA ticket.

### Checklist
- [ ] Code compiles correctly
- [ ] Created tests which fail without the change (if possible)
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: src/TopBar.js [hunk 1]
```diff
@@ -20,16 +20,17 @@ import { withStyles } from '@material-ui/core/styles';
...
   return (
     <AppBar className={classNames(classes.purpleGradient, classes.topLayer)}>
       <Toolbar>
         <Typography className={classes.title}>MOLTIMATE</Typography>
         <div className={classes.wide}>
-          <IconButton className={classes.floatRight} onClick={() => console.log('temp')}>
+          <IconButton className={classes.floatRight} onClick={() => toggleSettings()}>
             <SettingsIcon className={classNames(classes.floatRight, classes.white)}/>
           </IconButton>
+          <input type="file" onChange={(e) => uploadPDBQT(e.target.files)} multiple/>
```
[the remark this answers is not in the document]
Comment (kdm5498) [reply src/TopBar.js:33]: This is a temporary button for testing the component, resolving.

**Comment by kdm5498** [comment]
Git has decided that everything done by Eric is also part of this review. The parts actually under review are DummyData.js, MoltimateContainer.js, TopBar.js, ngl-util.js, and DockingContainer.js.

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Dependency directories
 node_modules/
 
+# Distribution files
```
[the remark this answers is not in the document]
Comment (kdm5498) [reply .gitignore:4]: I have added build instructions to the README file.

**Comment by ada-phillips** [comment]
It looks like you branched off of Eric's branch rather than dev, which is why all of the extra changes and merge conflicts got dragged in there. Would it be possible for you to rebase to `dev`, or just open a new PR using the diff between this one and dev? 

I really don't like how messy this is.

**Comment by kdm5498** [comment]
I just attempted a rebase to dev. It did not help at all with cleaning up, and I think it just made it messier.

**Comment by kdm5498** [comment]
Closed to open pull request on branch with only differences from dev.

**Pull request closed without merging**